src/libxdata/XData.cpp

```diff
@@ -66,7 +67,21 @@ void XData::cancelLoading() {
 }
 
 void XData::loadAirports() {
-    AirportLoader loader(world);
+    const AirportLoader loader(world);
+
+    //Custom scenery

```

Comment:
There should be a space after the comment mark and the word "scenery" is misspelled.